src/ProjectTemplates/GenerateTemplateContent/GenerateTemplateContent.csproj

```diff
@@ -1,9 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="$(MSBuildThisFileDirectory)..\GeneratedContent.props" />
-
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <SuppressFinalPackageVersion>true</SuppressFinalPackageVersion>
```

Comment:
This won't be needed with #6041

Reply:
Do you have a sense of when that PR will be merged? If that PR merges before this one, I'll remove this line.

Reply:
Once @joperezr approves it.

This is a non-blocking comment, if this PR goes in first, we just need to remember to undo this later.

Reply:
Let's address this in a follow-up. I'm going to go ahead and merge this now.